aip/general/0121.md

```diff
@@ -61,6 +61,22 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+If the request to or the response from a standard method (or a custom method in
+the same *service*) **is** the resource or **contains** the resource, the
+resource schema for that resource across all methods **must** be the same.
+
+| Standard method | Request               | Response        |
+| --------------- | --------------------- | --------------- |
+| Create          | Contains the resource | Is the resource |
+| Get             | None                  | Is the resource |
+| Update          | Contains the resource | Is the resource |
+| Delete          | None                  | None            |
+| List            | None                  | Is the resource |
+
+*The table above describes each standard method's relationship to the resource,
```

Comment:
Nit: Do you mean for this to be a "Note"? If So, use the `**Note:**` syntax, like above instead of italicizing the entire block.

Reply:
This was intentional, more as a footnote for the table rather than a **note:**, which stylistically seems to be for clarification about an AIP itself.